WordPress/src/main/res/layout/media_picker_fragment.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/media_empty_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:textSize="@dimen/empty_view_text_size"
+        android:gravity="center"
+        android:layout_gravity="center" />
+
+    <GridView
+        android:id="@+id/media_adapter_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:padding="4dp"
```

Comment:
Minor nit: it would be good to use `@dimen/margin_small` and `@dimen/margin_extra_small` here instead of hard-coding `4dp` and `2dp`.